doc/Overview/Mvux/Advanced/Selection.md

```diff
@@ -0,0 +1,137 @@
+It has a service which has the following contract:
+
+```csharp
+public interface IPeopleService
+{
+    ValueTask<IImmutableList<Person>> GetPeopleAsync(CancellationToken ct);
+}
+```
+
+It is then used by the `PeopleModel` class which requests the service using a List-Feed.
+
+```csharp
+public partial record PeopleModel(IPeopleService PeopleService)
+{
+    public IListFeed<Person> People => ListFeed.Async(PeopleService.GetPeopleAsync);
+}
+```
+
+The data is then displayed on the View using a `ListView`:
+
+```xml
+<Page ...>
+    <ListView ItemsSource="{Binding People}">
+        <ListView.ItemTemplate>
+            <DataTemplate>
+                <StackPanel Orientation="Horizontal" Spacing="5">
+                    <TextBlock Text="{Binding FirstName}"/>
+                    <TextBlock Text="{Binding LastName}"/>
+                </StackPanel>
+            </DataTemplate>
+        </ListView.ItemTemplate>
+    </ListView>
+</Page>
+```
+
+> [!NOTE]  
+> The use of the `FeedView` is not necessary in our example, hence the `ListView` has been extracted from it, and its `ItemsSource` property has been directly data-bound to the Feed.
+
+### Implement selection in the *PeopleApp*
+
+MVUX has two extension methods of `IListFeed<T>`, that enables single or multi selection.
+
+> [!NOTE]  
+> The source-code for the sample app demonstrated in this section can be found [here](https://github.com/unoplatform/Uno.Samples/tree/master/UI/MvuxHowTos/AdvancedPeopleApp).
+
+## Single-item selection
+
+A Feed doesn't store any state, so the `People` property won't be able to hold any information, nor the currently selected item.  
+To enable storing the selected value in the model, we'll create an `IState<Person>` which will be updated by the `Selection` operator of the `IListFeed<T>` (it's an extension method).
+
+Let's change the `PeopleModel` as follows:
+
+```csharp
+public partial record PeopleModel(IPeopleService PeopleService)
+{
+    public IListFeed<Person> People =>
+        ListFeed
+        .Async(PeopleService.GetPeopleAsync)
+        .Selection(SelectedPerson);
+
+    public IState<Person> SelectedPerson => State<Person>.Empty(this);
+}
+```
+
+The `SelectedPerson` State is initialized with an empty value using `State<Person>.Empty(this)` (we still need a reference to the current instance to enable caching).
+
+> [!NOTE]  
+> Read [this](xref:Overview.Mvux.States#other-ways-to-create-feeds) to learn more about States and the `Empty` factory method.
+
+The `Selection` operator was added to the existing `ListFeed.Async(...)` line, it will listen to the `People` List-Feed, and will affect its selection changes onto the `SelectedPerson` State property.
+
+In the View side, wrap the `ListView` element in a `StackPanel`, and insert additional elements to reflect the currently selected value via the `SelectedPerson` State.  
```

Comment:
Don't **ever** do this - you should never put a ListView inside a StackPanel. Please put it inside a Grid with two rows

Reply:
@nickrandolph the example has also been updated, please review: https://github.com/unoplatform/Uno.Samples/pull/334